ui/k-info-panel/switch_info.kytos

```diff
@@ -27,6 +28,11 @@
    props: ["content"],
    data () {
      return {
+       nextState: '',
```

Comment:
@Alopalao I hate to have to ask this, since this is supposed to be a job of a linter / codestyle chck but since we don't have one yet, could you please use snake case variables? `next_state` and `is_active`? Typically, in JavaScript folks use camel case as you've done here, however the code base has been sticking to snake case.

Reply:
So sorry about that, I noticed it but I got mixed with other project. Fixed.

Reply:
So sorry about that, I noticed it but I got mixed with other project. Fixed. Commit [4e91ab3ee60976a1d6fabfb1cb041793c253b9ed](https://github.com/kytos-ng/topology/pull/70/commits/4e91ab3ee60976a1d6fabfb1cb041793c253b9ed)